tests/topotests/lib/bgp.py

```diff
@@ -86,6 +97,7 @@ def create_router_bgp(tgen, topo, input_dict=None, build=False):
                                             "allowas-in": {
                                                     "number_occurences":2
                                             },
+                                            "graceful-restart": True",
```

Comment:
looks like an error here

Reply:
This example data in framework API. It will guide users, how to call create_router_bgp(). This data will not be used anywhere.

Reply:
Yes, but copy/pasting introduces a JSON error :)